README.md

```diff
@@ -14,19 +14,19 @@ Desefa is a game created and developed by Victor Gaudiot, Pedro Arruda, João Pe
 ## About the game
 This game was created as a project to "Game development" class. Our great inspiration is chess and decided to add some new elements to make it more interesting.
 ## How to play
-Mostly like normal chess.
+Mostly like normal chess, pieces will move in the same way, you can check for [pieces movements](#chess-pieces) on the next section. But, be careful, you may find some obstacles on the way, and that will interrupt your movement, just like attacking a oponents piece. That interrupts your turn and its time for the other player do it's play. 
 ## Chess pieces
 
  - Pawn♙
-	 -  
+	 -  Each player starts the game with 8 pawns, filling the front line of pieces. They always move foward on vertical direction, one square at a time, except the first move of each, when you can decide if you go one or two squares ahead. Pawns attack are also realized moving foward, but in diagonal direction.
```

Comment:
Make two bullet points.
- Piece
    - Quantity at start
    - Movement description

Reply:
change **foward** to **forward**